packages/apps/graph-client/src/pages/index.tsx

```diff
@@ -1,89 +1,54 @@
 import { useGetBlocksSubscription } from '../__generated__/sdk';
-import styles from '../styles/Home.module.css';
+import { ChainwebGraph } from '../components/chainweb';
+import { Text } from '../components/text';
+import { styled } from '../styles/stitches.config';
+import { useParsedBlocks } from '../utils/hooks/useParsedBlocks';
 
 import Head from 'next/head';
-import Image from 'next/image';
-import React, { useEffect, useState } from 'react';
+import React, { useEffect } from 'react';
+
+const StyledMain: any = styled('main', {
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'center',
+  flexDirection: 'column',
+  my: '5rem',
+});
 
 export default function Home(): JSX.Element {
   const { loading, data } = useGetBlocksSubscription();
   const dataString = JSON.stringify(data);
-  const [allBlocks, setBlocks] = useState<any[]>([]);
+
+  const { allBlocks, addBlocks } = useParsedBlocks();
 
   useEffect(() => {
-    setBlocks((b) => [...b, ...(data?.newBlocks.map((b) => b) ?? [])]);
+    if (data?.newBlocks && data?.newBlocks?.length > 0) {
+      addBlocks(data?.newBlocks);
+    }
   }, [dataString]);
 
   return (
-    <div className={styles.container}>
+    <div>
       <Head>
-        <title>Create Next App</title>
-        <meta name="description" content="Generated by create next app" />
-        <link rel="icon" href="/favicon.ico" />
+        <title>Kadena Graph Client</title>
+        <link
+          rel="icon"
+          href="https://kadena.io/wp-content/uploads/2021/10/Favicon-V1.png"
```

Comment:
Can we add this file to this repo? Or is this SO temporary/internal-use that it doesn't really matter if this file doesn't exist anymore?

Reply:
No, this is just internal. To have something to show how the graph is consumed. It's all very PoC